src/clamp.js

```diff
@@ -0,0 +1,27 @@
+var _curry3 = require('./internal/_curry3');
+
+/**
+ * Clamps a number within a range.
```

Comment:
Shall we mention that this function applies to any type which supports ordering?